<launch>
  <arg name="use_fake_hardware" default="true" description="Use fake hardware (true | false)" />

  <group if="$(var use_fake_hardware)">
    <include file="$(find-pkg-share franka_moveit_config)/launch/moveit.launch.py" >
      <arg name="robot_ip" value="dont-care"/>
      <arg name="use_fake_hardware" value="true"/>
    </include>
  </group>

  <group unless="$(var use_fake_hardware)">
    <include file="$(find-pkg-share franka_moveit_config)/launch/rviz.launch.py" >
      <arg name="robot_ip" value="panda0.robot"/>
      
    </include>
    <include file="$(find-pkg-share drawing)/april_tag.launch.xml"></include>
  </group>
  
  <node pkg="drawing" exec="draw" name="Drawing">
    <param name="use_fake_hardware" value="$(var use_fake_hardware)"/>
    <param name="robot_name" value="panda"/>
    <param name="group_name" value="panda_manipulator"/>
    <param name="frame_id" value="panda_link0"/>
  </node>

  <node pkg="drawing" exec="executor" name="Execute"/>
  
</launch>